examples/defaulter-gen/generators/defaulter.go

```diff
@@ -731,6 +803,26 @@ func (n *callNode) writeCalls(varName string, isVarPointer bool, sw *generator.S
 	}
 }
 
+func (n *callNode) writeDefaulter(varName string, isVarPointer bool, sw *generator.SnippetWriter) {
+	accessor := varName
+	if !isVarPointer {
+		accessor = "&" + accessor
+	}
+	if n.defaultValue != "" {
+		sw.Do("if reflect.ValueOf($.var$).IsNil() {\n", generator.Args{
+			"var": varName,
+		})
+		sw.Do("if err := json.Unmarshal([]byte(`$.defaultValue$`), $.var$); err != nil {\n", generator.Args{
```

Comment:
Don't you miss an `&` before `$.var$`?

Reply:
Ah, no ok I can see the accessor above, cool.

Reply:
Maybe use a different name "var" is a little confusing

Reply:
renamed to varPointer